test/e2e/density.go

```diff
@@ -210,6 +214,7 @@ var _ = Describe("Density", func() {
 				PodStatusFile:        fileHndl,
 				Replicas:             totalPods,
 				MaxContainerFailures: &MaxContainerFailures,
+				Timeout:              time.Duration(math.Ceil(float64(itArg.podsPerNode)/minPodStartupThroughput)) * time.Minute,
```

Comment:
I agree node is not the only bottleneck in a larger cluster. However, the current timeout (if not set) by default is just 5 min. This may not even be enough for a small cluster on a bad day. We *need* to set a reasonable timeout anyway, and the test is not setting one based on density and/or cluster size.